packages/vite/src/node/server/middlewares/error.ts

```diff
@@ -82,7 +84,7 @@ export function errorMiddleware(
                 '\\u003c',
               )}
               try {
-                const { ErrorOverlay } = await import('/@vite/client')
+                const { ErrorOverlay } = await import(${path.posix.join(server.config.base, CLIENT_PUBLIC_PATH)})
```

Comment:
This is missing quotes around the specifier, you can wrap it with a `JSON.stringify()` so that it generates quotes and escape them.

Reply:
ok. thanks